README.md

```diff
@@ -1,11 +1,86 @@
-> [!WARNING] 
-> We are no longer recommending this SDK or Wasm in general for anyone due to the fundamental memory issue of TinyGo (See [the detailed explanation](https://github.com/tetratelabs/proxy-wasm-go-sdk/issues/450#issuecomment-2253729297) by a long-time community member)
-> as well as [the project state of Proxy-Wasm in general](https://github.com/envoyproxy/envoy/issues/35420).
-> If you are not in a position where you have to run untrusted binaries (like for example, you run Envoy proxies while your client gives you the binaries to run), we recommend using other extension mechanism 
-> such as Lua or External Processing which should be comparable or better or worse depending on the use case. 
-> 
-> If you are already using this SDK, but still want to continue using Wasm for some reason instead of Lua or External Processing, 
-> we strongly recommend migrating to the Rust or C++ SDK due to the memory issue of TinyGo described in the link above.
-> 
-> We keep this repository open and not archived for the existing users, but we cannot provide any support or guarantee for the future development of this SDK.
-> However, at any time, we may decide to archive this repository if we see no reason to keep it open.
+# WebAssembly for Proxies (Go SDK) [![Build](https://github.com/tetratelabs/proxy-wasm-go-sdk/workflows/Test/badge.svg)](https://github.com/tetratelabs/proxy-wasm-go-sdk/actions) [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](LICENSE)
+
+The Go SDK for
+ [Proxy-Wasm](https://github.com/proxy-wasm/spec), enabling developers to write Proxy-Wasm plugins in Go. 
+
+## Getting Started
+
+- [examples](examples) directory contains the example codes on top of this SDK.
+- [OVERVIEW.md](doc/OVERVIEW.md) the overview of Proxy-Wasm, the API of this SDK, and the things you should know when writing plugins.
+
+## Requirements
+
+- \[Required] [Go](https://go.dev/): v1.24+ - This SDK leverages Go 1.24's support for [WASI](https://github.com/WebAssembly/WASI) (WebAssembly System Interface) reactors. You can grab a release candidate from the [Go unstable releases page](https://go.dev/dl/#unstable).
+- \[Optional] [Envoy](https://www.envoyproxy.io) - To run end-to-end tests, you need to have an Envoy binary. You can use [func-e](https://func-e.io) as an easy way to get started with Envoy or follow [the official instruction](https://www.envoyproxy.io/docs/envoy/latest/start/install).
+
+## Installation
+
+```
+go get github.com/tetratelabs/proxy-wasm-go-sdk
```

Comment:
Update! :-)

Please search for other instances e.g. import statements as in the example below.